### DISCUSSION TITLE
[Project Beta][Bulk Add] Bulk remove from project

### DISCUSSION THREAD

**Discussion by schalkneethling** [community/community#14995] [Projects and Issues]
Thank you so much for adding the ability to add issues in bulk to a project. Two things I would like.

## Show project issue is a part of

As can be seen in the screenshot below, when viewing issues in the issue list, there is no indication that an issue was added to a specific project or projects. That would be really useful.

![Screenshot 2022-04-15 at 11 06 05](https://user-images.githubusercontent.com/10350960/163551143-fc3339b5-7a2a-4756-bc7b-3e3269d0bda9.png)

## Show assigned projects in project drop-down

When adding issues in bulk, the projects[Beta] dropdown does not show the current project(s) the selected issue(s) belong to. Having this will open the door for removing issues in bulk from a project, or event projects.

### Remember previous selections in project[beta] dropdown

When I have added some issues to a project using the new bulk add option, the dropdown does not remember the previous projects I have selected. While this is really a nice to have, I believe that when viewing a pull request, for example, the projects dropdown does remember a previously selected project. It also shows the current projects the PR belongs to.

**Comment by usirg** [comment]
Let me stress that the second point is really important, i.e. showing the selection state of issues in the project drop-down so that issues can be unlinked in bulk!

Imagine accidentally linking a hundred 100 issues to the wrong project in bulk and then having to unlink them individually. Not a good situation right now.
